src/components/UI/Charts/Sunburst/sunburst-d3.jsx

```diff
@@ -18,20 +18,11 @@ const sunburstD3 = (ref, data, config, formatters) => {
     .innerRadius((d) => (d.y1 <= 2 ? d.y0 * (radius + 28) : d.y0 * (radius + 2)))
     .outerRadius((d) => Math.max(d.y0 * radius, d.y1 * radius - 2));
 
-  // since results varies on each phenotype selection
-  // e.g. a parent can have more results than it's single child
-  // each level need to countain the concatenated sum of all its child
-  // to be use all the reprensentive radia space in the circle
-  const countResults = (d) => {
-    if (d.length === 0) return 0;
-    d.reduce((accumulator, value) => accumulator + value.results + countResults(value.children));
-  };
-
   const partition = (data) => {
     const root = d3
       .hierarchy(data)
-      .sum((d) => d.results + countResults(d.children))
-      .sort((a, b) => b.results - a.results);
+      .sum((d) => d.value)
```

Comment:
out of curiosity, sum gives an array? I would have thought that .sum gives a scalar. Therefore, sorting would make no sense.

Reply:
sum is for the value of the element, sort is how to place them on the arc